src/index.js

```diff
@@ -241,6 +242,11 @@ function App() {
             <ParentSettings LoadingComponent={ParentLoadingComponent} />
           )}
         />
+        <Route
+          exact
+          path="/parent/changepassword"
+          component={() => <ChangePassword />}
+        />
```

Comment:
Should the component be rendered by ProtectedRoute instead? Right now you have Route to render ChangePassword.